whatlies/language.py

```diff
@@ -28,16 +28,20 @@ def _input_str_legal(string):
 
     def __getitem__(self, string):
         self._input_str_legal(string)
-        doc = self.nlp(string)
-        vec = doc.vector
+        bert = False
         start, end = 0, -1
         split_string = string.split(" ")
         for idx, word in enumerate(split_string):
             if word[0] == "[":
+                bert = True
                 start = idx
             if word[-1] == "]":
                 end = idx + 1
-        if start != 0:
-            if end != -1:
-                vec = doc[start:end].vector
-        return Embedding(string, vec)
+        if end == -1:
+            end = start + 1
+        if bert:
+            new_str = string.replace("[", "").replace("]", "")
+            return Embedding(string, self.nlp(new_str)[start].vector)
```

Comment:
i wonder why you only look at `start` here. 

these two examples should yield a different result; 

```
i am going to a [nice beach]
i am going to a [nice] beach
```